src/main/database/liquibase.changelog.xml

```diff
@@ -0,0 +1,32 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+    <changeSet id="schema_snoop" author="plin" failOnError="false">
```

Comment:
My concern would be that `failOnError=false` might mask any other sort of error that could happen (e.g. permissions).  In Mercury we usually did something like
```
    <changeSet ...
        <preConditions onFail="MARK_RAN">
            <not>
                <tableExists tableName="SUBMISSION"/>
            </not>
        </preConditions>
        <createTable ...
```
So Liquibase wouldn't try to create the table if it already existed, but the changeSet would be left in a "has been processed" state.